README.md

```diff
@@ -176,15 +175,17 @@ Chapter 14 - R6
 Chapter 15 - S4
 
 - Cohort 1: Pavitra Chakravarty ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week14/Cohort1/Chapter15.html#1) | [video](https://www.youtube.com/watch?v=a1jzpWiksyA&feature=youtu.be))
+- Cohort 2: Erick Knackstedt (SLIDES | [video](https://youtu.be/bzo37PHCM1I))
```

Comment:
Slides are openPR in #234 as well, integrate here?

Reply:
I think there’s a link to chapter four as well

Reply:
Yup, just came here to merge that PR in, sorry about that...

Reply:
It was chapter 5. Merged that PR into this one and resolved conflicts.